### PR TITLE
Track qr scans

### PR DESCRIPTION
**Pull request by stevenh17**
🎟️ **Ticket(s):**
Closes #

👷 **Changes:**
export const QR_SCANS_RECORD = "biztechQRScans";

A new table with the following fields
    1. timestamp as PK
    2. FK to QRCode scanned
    3. FK to user which scanned QRCode

Scanning a QR now calls a helper function to store QR and user information into biztechQRScans table

💭 **Notes:**
I made timestamp a string and not a number, but if that's a problem I can remove the toString()

image

image

**Wait! Before you merge, have you checked the following:**
- [ ] Serverless tests are passing (Check travis build logs, CI is currently broken)
- [ ] PR is has approving review(s)

### CODE REVIEW

File: services/qr/handler.js [hunk 1]
```diff
@@ -62,36 +62,29 @@ export const post = async (event, ctx, callback) => {
...
             message: "ERROR: " + res.errorMessage,
             response: res
           });
-
           callback(null, response_fail);
           return response_fail;
+        } else {
+          try {
+            await registrationHelpers.logQRScan(data.qrCodeID, data.email);
+            console.log("hey success!!");
```
Comment (jerryxu99) [inline services/qr/handler.js:76]: before submitting PRs, a good practice is to look at every line that you changed to make sure each line change is intended. 

Let's make these console logs more meaningful

File: services/qr/handler.js [hunk 2]
```diff
@@ -62,36 +62,29 @@ export const post = async (event, ctx, callback) => {
...
-          ) {
-            const response_fail = helpers.createResponse(406, {
-              message: "ERROR: " + res.errorMessage,
-              response: res
-            });
-
-            callback(null, response_fail);
-            return response_fail;
-          }
-
```
[the remark this answers is not in the document]
Comment (stevenh17) [reply services/qr/handler.js:78]: oops deleted by accident i added it back

File: services/qr/handler.js (hunk 1 above)
Comment (stevenh17) [reply services/qr/handler.js]: ok i removed the debugging console logs

**Review by ddennis924 — COMMENTED** [review]
have merge conflicts

**Comment by stevenh17** [comment]
> have merge conflicts

there was one line dev had that my branch didn't, but i think i resolved it

**Review by ddennis924 — APPROVED** [review]

**Pull request merged**